internal/pipe/checksums/checksums.go

```diff
@@ -55,6 +56,22 @@ func (Pipe) Run(ctx *context.Context) (err error) {
 		return nil
 	}
 
+	extraFiles, err := extrafiles.Find(ctx.Config.Checksum.ExtraFiles)
+	if err != nil {
+		return err
+	}
+
+	for name, path := range extraFiles {
+		if _, err := os.Stat(path); os.IsNotExist(err) {
+			return fmt.Errorf("failed to checksum %s: %w", name, err)
+		}
```

Comment:
this check is not needed I think, the `extrafiles.Find` would guarantee the file exists...